heartbeat/mysql

```diff
@@ -719,13 +719,22 @@ mysql_monitor() {
     fi
  
     mysql_common_status $status_loglevel
-
     rc=$?
 
     # TODO: check max connections error
 
     # If status returned an error, return that immediately
     if [ $rc -ne $OCF_SUCCESS ]; then
+        if ( ocf_is_ms ); then
```

Comment:
No particular reason. Just a code style I follow.  So I better resubmit this request without the ()s?